test/controllers/projects/automated_workflow_executions_controller_test.rb

```diff
@@ -0,0 +1,157 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+module Projects
+  class AutomatedWorkflowExecutionsControllerTest < ActionDispatch::IntegrationTest
+    include Devise::Test::IntegrationHelpers
+
+    test 'can get the listing of automated workflow executions for a project' do
+      sign_in users(:john_doe)
+
+      namespace = groups(:group_one)
+      project = projects(:project1)
+
+      get namespace_project_automated_workflow_executions_path(namespace, project)
+
+      assert_response :success
+    end
+
+    test 'can create a automated workflow execution for a project' do
+      sign_in users(:john_doe)
+
+      namespace = groups(:group_one)
+      project = projects(:project2)
+
+      post namespace_project_automated_workflow_executions_path(namespace, project),
+           params: { automated_workflow_execution: {
+             metadata: { workflow_name: 'phac-nml/iridanextexample', workflow_version: '1.0.2' },
+             workflow_params: { assembler: 'stub' },
+             email_notification: true,
+             update_samples: true
+           }, format: :turbo_stream }
+
+      assert_response :success
```

Comment:
**nitpick (blocking):** could you add a test to check that the newly created `AutomatedWorkflowExecution` has the attributes set as specified. This would have caught the `updated_samples` bug from above.